torchci/lib/bot/ciflowPushTrigger.ts

```diff
@@ -1,5 +1,5 @@
 import { Context, Probot } from "probot";
-import { CachedConfigTracker, isPyTorchPyTorch } from "./utils";
+import { CachedConfigTracker, isPyTorchPyTorch, repoKey } from "./utils";
```

Comment:
Nit: I don't see `repoKey` is used anywhere, so it could be removed?

Reply:
thanks for the catch :)